processor/src/main/resources/org/mapstruct/ap/internal/model/common/Parameter.ftl

```diff
@@ -19,4 +19,4 @@
      limitations under the License.
 
 -->
-<@includeModel object=type/> ${name}
\ No newline at end of file
+<@includeModel object=type allowVarArgs=ext.allowVarArgs!false/> ${name}
```

Comment:
Does the `!` in `ext.allowVarArgs!false` mean if it is not set use `false`? Like default?

Reply:
Exactly, otherwise Freemarker is complaining when `allowVarArgs` is not available